Source/MOLCertificate.m

```diff
@@ -312,19 +312,36 @@ - (NSString *)SHA1 {
 }
 
 - (NSString *)SHA256 {
-  return [self memoizedSelector:_cmd forBlock:^id{
-    NSMutableData *SHA256Buffer = [[NSMutableData alloc] initWithCapacity:CC_SHA256_DIGEST_LENGTH];
-
-    CC_SHA256([self.certData bytes], (CC_LONG)[self.certData length], [SHA256Buffer mutableBytes]);
-
-    const unsigned char *bytes = (const unsigned char *)[SHA256Buffer bytes];
-    NSMutableString *hexDigest = [NSMutableString stringWithCapacity:CC_SHA256_DIGEST_LENGTH * 2];
-    for (int i = 0; i < CC_SHA256_DIGEST_LENGTH; i++) {
-      [hexDigest appendFormat:@"%02x", bytes[i]];
-    }
-
-    return hexDigest;
-  }];
+  return
+      [self memoizedSelector:_cmd
+                    forBlock:^id {
```

Comment:
If you put this back as:

```objc
return [self memoizedSelector:_cmd forBlock:^id {
```

Then the block doesn't need to be indented as far and will be easier to read.